### DISCUSSION TITLE
mooseInfo question

### DISCUSSION THREAD

**Discussion by jessecarterMOOSE** [idaholab/moose#20932] [Q&A General] [answered]
Is there a way for a message sent to `mooseInfo()` to output each time it's called? Looks like it's wrapped in a "mooseDoOnce" call which I assume does what it says?

For instance, I'm using a Terminator to cut the time step when a material model needs it (using `MaterialTimeStepPostprocessor`) and I'd like it to print the Terminator message each time rather than just the first time.

**Comment by GiudGiud** [comment, marked as answer]
Hello

Not currently. We have a PR up to improve that system and be able to output things on a customizable schedule. 
https://github.com/idaholab/moose/pull/17897

Guillaume

**Reply by jessecarterMOOSE** [reply]
Thanks. Looks like that PR is a bit old but I hope it gets through because the functionality would be useful.

**Reply by GiudGiud** [reply]
If this PR doesnt make it in I ll make another one next week. We have a few users for this functionality at this point.

**Reply by GiudGiud** [reply]
We're still working on it. I ll try to get you an update on this next week

**Reply by GiudGiud** [reply]
this PR is stalled as its makers arent happy with it. I have another idea for this but will take some free time.

**Reply by GiudGiud** [reply]
btw we're not going to change mooseInfo's behavior. Use a _console output for your needs

**Reply by jessecarterMOOSE** [reply]
Got it, thanks.
